app/src/main/res/layout/nav_header_navigation_drawer.xml

```diff
@@ -1,48 +1,56 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto">
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:profile="http://schemas.android.com/tools">
 
   <data>
+
     <variable
       name="viewModel"
       type="org.oppia.app.drawer.NavigationDrawerHeaderViewModel" />
   </data>
 
   <LinearLayout
+    android:id="@+id/header_linear_layout"
     android:layout_width="match_parent"
     android:layout_height="176dp"
     android:background="@drawable/side_nav_bar"
     android:gravity="bottom"
+    android:onClick="@{(v) -> viewModel.onHeaderClicked()}"
     android:orientation="vertical"
     android:paddingStart="@dimen/activity_horizontal_margin"
     android:paddingTop="@dimen/activity_vertical_margin"
     android:paddingEnd="@dimen/activity_horizontal_margin"
     android:paddingBottom="@dimen/activity_vertical_margin"
     android:theme="@style/ThemeOverlay.AppCompat.Dark">
 
-    <ImageView
-      android:layout_width="64dp"
-      android:layout_height="64dp"
-      android:contentDescription="@string/nav_header_desc"
+    <de.hdodenhof.circleimageview.CircleImageView
+      android:id="@+id/profile_list_image"
```

Comment:
We can simple name it as `profile_image` instead of `profile_list_image`.

Reply:
We can actually just remove this ID directly as it is not getting used anywhere.